src/streaming-services/nrk/NrkApi.ts

```diff
@@ -140,7 +140,11 @@ class _NrkApi extends Api {
 				});
 				const responseJson = JSON.parse(responseText) as NrkProgressResponse;
 				const { progresses } = responseJson;
-				this.HISTORY_API_URL = this.API_HOST_URL + responseJson._links.next.href;
+				if (responseJson._links.next) {
```

Comment:
Right, I hadn't thought of checking for the next URL to make sure we're on the last page. So we don't need to store the progress IDs after all.